.sops.yaml

```diff
@@ -78,26 +76,20 @@ creation_rules:
           # - *age_r-vdp
           - *mamading_age
           - *age_thetasinner
-        pgp:
-          - *steveej
```

Comment:
Don't want to deal with GPG for one user who also had an age key but also don't see a reason to encrypt the new secret towards this user's key.